index.html

```diff
@@ -950,11 +956,257 @@ <h3>
-          credentials from individuals such as requesting a driver's license to
-          log into a movie rating website and how the ecosystem can mitigate
-          this risk.
+        <p class="issue" title="Work in progress"></p>
+        <p>
+          Unnecessary credential requests are a key privacy risk of the entire
+          digital credentials ecosystem. They could manifest in different ways
+          and from different motivations:
+        </p>
+        <ul>
+          <li>Intentional abuse of the API to learn sensitive information about
+          the user for the purpose of fraud, tracking, or sale of the data. For
+          example, a site could trick a user into sharing their passport
+          information through misleading content. This can lead to identity
+          theft and financial loss, and severe loss of control and/or leakage
+          of personal information.
+          </li>
+          <li>Unnecessary requests for credentials without the explicit intent
+          of user harm, such as an online store requesting users to sign up
+          with their driver's license instead of generic email & password or
+          federated credentials. This can lead to exclusion of users without
+          the ability or willingness to share such a credential with the site,
+          a deterioration of the prompt experience on the web, and an increase
+          in the risk of accidental data leakage.
+          </li>
+          <li>Requests for an excessive amount of information for valid
+          purposes against the principle of data minimization. A common example
+          is collection of a user's entire national identity document for age
+          verification instead of relying on selective disclosure.
+          </li>
+        </ul>
+        <p>
+          One challenge here is determining what constitutes "valid" purposes
+          and which requests are therefore "unnecessary", and requires
+          participation from all parties involved in the credential exchange.
+        </p>
+        <ul>
+          <li>Ideally, verifiers would self-regulate their requests for
+          credentials. However, from a user agent's perspective, verifiers are
+          potential attackers, and might not consider the user's best interest
+          in their designs.
+          </li>
+          <li>User agents are responsible for protecting their users against
+          dangerous content and permission requests on the Web and could
+          intervene on their behalf, proactively rejecting requests.
+          </li>
+          <li>Issuers and lawmakers might decide to restrict use of
+          (particularly government-issued) credentials to specific verifiers
+          with purpose attestations. Wallets might be expected to enforce these
+          restrictions.
+          </li>
+          <li>The ultimate decision of whether or not to share their personal
```

Comment:
We should again say, "by design" ... the spec requires a UI to be presented (or it will... that will be a MUST).